src/app/admin/product/components/product-color/product-color.component.html

```diff
@@ -9,10 +9,11 @@
       <div fxLayout="row" fxLayout.xs="column" fxLayoutAlign="start start"
```

Comment:
there is no function for modifyColor() in typeScript file..please add it or remove (ngSubmit)="modifyColor()" from line 8